modules/database/base/base.py

```
@@ -0,0 +1,192 @@
+    def _get_first_entry(self, table_schema, *filters) -> None | object:
+        query = self._build_query(table_schema, *filters)
+
+        return query.first()
+
+    def _get_all_entries(self, table_schema, *filters) -> List[object]:
+        query = self._build_query(table_schema, *filters)
+
+        return query.all()
+
+    def _get_attribute_from_first_entry(self, table_schema, field_name, *filters) -> None | object:
+        entry = self._get_first_entry(table_schema=table_schema, *filters)
```

Ruff (0.8.0)

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)